example/pubspec.yaml

```diff
@@ -18,7 +18,7 @@ publish_to: 'none' # Remove this line if you wish to publish to pub.dev
 version: 1.0.0+1
 
 environment:
-  sdk: ">=2.7.0 <3.0.0"
+  sdk: ">=2.12.0 <3.0.0"
```

Comment:
Is this necessary? Does it mean for example Flutter 1.22 will not be able to use this after the upgrade?

Reply:
No no. I should make it compatible with lower version.